### DISCUSSION TITLE
Transaction modes for PostgreSQL adapter

### DISCUSSION THREAD

**Discussion by runeflobakk** [tconbeer/harlequin#606] [Q&A, Troubleshooting, Support] [answered]
In the documentation it is described that [v0.3 of harlequin-postgres supports two transaction modes](https://harlequin.sh/docs/postgres/transactions). Are there any plans to add this support to harlequin-postgres, which is currently at [v0.2.2](https://github.com/tconbeer/harlequin-postgres/releases/tag/v0.2.2)?

I could try dipping my toes and make a contribution to add support for this, though I am not that familiar with actual python development. Are there any existing implementation of this in other adapters that would be applicable to use as a "cheat sheet"?

**Comment by tconbeer** [comment, marked as answer]
🤦 I forgot to merge my local branch and cut a new release. Sorry. Doing that now.

**Comment by runeflobakk** [comment]
Thanks! :) 

Just adding this for reference in case anybody else also struggle (as I was) to figure out how to upgrade an _injected package_ with **pipx**:
```sh
pipx upgrade --include-injected harlequin
```
https://fig.io/manual/pipx/upgrade
